spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -146,6 +153,31 @@ data class SpdxCompoundExpression(
 ) : SpdxExpression() {
     override fun decompose() = left.decompose() + right.decompose()
 
+    override fun disjunctiveNormalForm(): SpdxExpression {
+        val leftDnf = left.disjunctiveNormalForm()
+        val rightDnf = right.disjunctiveNormalForm()
+
+        return when (operator) {
+            SpdxOperator.OR -> SpdxCompoundExpression(leftDnf, SpdxOperator.OR, rightDnf)
+            SpdxOperator.AND -> {
```

Comment:
Nit: You could reduce the nesting by writing `SpdxOperator.AND -> when {`. Maybe also add a blank line before this one, even if the previous case is not a block?

Reply:
Done.